<html>
<script>

// A try...catch statement that throws a built-in error
const fixedString = 'Cannot be reassigned';
try{ 
 fixedString = 'A new string'; // A TypeError will be thrown  
}catch(error){
}
console.log("ankit is a good boy");
console.log('Prints after error'); // 
</script>
</html>